scripts/new_project.kts

```diff
@@ -34,18 +51,102 @@ object NewProject {
         }
     }
 
+    private fun repackage() {
+        showMessage("=> 🔎 Rename the package folders...")
+        modules.forEach { module ->
+            val srcPath = newProjectPath + File.separator + module + File.separator + "src"
+            for (file in File(srcPath).listFiles()) {
```

Comment:
@Tuubz unfortunately we cannot use `walk` in this case since `walk` will loop and find all nested directories in the module, but we only need the directories inside `src` folder 😅

Reply:
@Tuubz but what if there is more than a folder named "java" in our project? It could be a false case 😄  Right now I just want to update the folder `src/[main/staging/production]/java`, the other cases like `src/[main/staging/production]/java/.../java` is not the expected result.
Let's try to test by creating a `java` folder, and the result is:
```
walk /Users/tranmanh/Desktop/nimble/github/android-templates/TestProject/app/src/test/java
walk /Users/tranmanh/Desktop/nimble/github/android-templates/TestProject/app/src/main/java
walk /Users/tranmanh/Desktop/nimble/github/android-templates/TestProject/app/src/main/java/co/nimblehq/coroutine/extension/java
walk /Users/tranmanh/Desktop/nimble/github/android-templates/TestProject/app/src/debug/java
walk /Users/tranmanh/Desktop/nimble/github/android-templates/TestProject/data/src/test/java
walk /Users/tranmanh/Desktop/nimble/github/android-templates/TestProject/data/src/main/java
walk /Users/tranmanh/Desktop/nimble/github/android-templates/TestProject/domain/src/test/java
walk /Users/tranmanh/Desktop/nimble/github/android-templates/TestProject/domain/src/main/java
```
Yeah, `/Users/tranmanh/Desktop/nimble/github/android-templates/TestProject/app/src/main/java/co/nimblehq/coroutine/extension/java` is not the folder we want to update 🤪

Reply:
@manh-t Oh, you're right

In that case, we can use `maxDepth(2)` 😁

Reply:
@Tuubz that makes sense now 🎉